DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/SelfAssessmentOverview.cshtml

```diff
@@ -1,103 +1,104 @@
+    Manage optional @Model.VocabPlural().ToLower()
+  </a>
+}
+@if (Model.SelfAssessment.IsSupervised) {
+  <a class="nhsuk-button nhsuk-button--secondary trigger-loader" asp-route-selfAssessmentId="@Model.SelfAssessment.Id" asp-action="StartRequestVerification">
+    Request @Model.SelfAssessment.Vocabulary.ToLower() verification
+  </a>
+}
+@if (Model.CompetencyGroups.Any()) {
+  foreach (var competencyGroup in Model.CompetencyGroups) {
+    <div class="nhsuk-panel competency-group-panel nhsuk-u-padding-0">
+      <details class="nhsuk-details nhsuk-expander" @(Model.CompetencyGroups.Count() == 1 ? "open" : ViewContext.RouteData.Values.ContainsKey("competencyGroupId") ? ViewContext.RouteData.Values["competencyGroupId"].Equals(competencyGroup.First().CompetencyGroupID.ToString()) ? "open" : "" : "")>
+        <summary class="nhsuk-details__summary">
+          <span class="competency-group-title nhsuk-details__summary-text">
+            @competencyGroup.Key
+          </span>
+        </summary>
+        <div class="nhsuk-details__text">
+          @if (Model.SelfAssessment.LinearNavigation) {
+            <partial name="SelfAssessments/_MeanScores"
+                     view-data="@(new ViewDataDictionary(ViewData) { { "competencyGroup", competencyGroup } })" />
+          }
+
+          <partial name="SelfAssessments/_OverviewTable" view-data="@( new ViewDataDictionary(ViewData) { { "linearNavigation", Model.SelfAssessment.LinearNavigation }, { "selfAssessment", Model.SelfAssessment } })" model="competencyGroup" />
+
+        </div>
+      </details>
+      @if (Model.SelfAssessment.LinearNavigation) {
+        <div class="outer-score-container nhsuk-u-padding-bottom-4">
+          <partial name="SelfAssessments/_MeanScores"
+                   view-data="@(new ViewDataDictionary(ViewData) { { "competencyGroup", competencyGroup } })" />
+        </div>
+      }
+    </div>
+  }
+}
+
+@if (Model.SelfAssessment.IncludesSignposting) {
+  <p class="nhsuk-u-reading-width">Once you are happy with your responses, submit your self-assessment to retrieve a list of recommended learning resources.</p>
+
+  @if (Configuration.IsSignpostingUsed()) {
+    <a class="nhsuk-button finish-review-button trigger-loader"
+       asp-controller="RecommendedLearning"
+       asp-route-selfAssessmentId="@Model.SelfAssessment.Id"
+       asp-action=@(Model.SelfAssessment.UnprocessedUpdates ? "SelfAssessmentResults" : "RecommendedLearning")>
+      Submit results
+    </a>
+  } else {
+    <a class="nhsuk-button finish-review-button trigger-loader"
+       asp-controller="RecommendedLearning"
+       asp-route-selfAssessmentId="@Model.SelfAssessment.Id"
+       asp-action=@(Model.SelfAssessment.UnprocessedUpdates ? "SelfAssessmentResults" : "RecommendedLearning")>
+      Submit self assessment
+    </a>

```

Comment:
role=button?

Reply:
I've added them to all links with class nhsuk-button in this view